ui/app/templates/partials/replication/update-primary.hbs

```diff
@@ -6,7 +6,7 @@
     </p>
     <div class="field">
       <label for="dr_operation_token" class="is-label">
-        DR Operation Token
+        DR Operation token
```

Comment:
Would the O be o ?
```suggestion
        DR operation token
```

Reply:
I think so (I may have just missed that)